.github/workflows/beta.yml

```diff
@@ -37,7 +37,27 @@ jobs:
       
     - name: Build with Gradle
       run: ./gradlew build
-      
+
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: __token__
+        TWINE_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
+      run: |
+        cd dist
+        cd python
+        python setup.py sdist bdist_wheel
+        twine upload dist/*
+
```

Comment:
move this to build release... just the 1.17 one tho...

Reply:
Has been done.